src/app/container/container.component.ts

```diff
@@ -197,8 +197,15 @@ export class ContainerComponent extends Entry implements AfterViewInit {
     if (tool) {
       this.tool = tool;
       this.initTool();
+      this.requestAccessHREF$ = this.extendedTool$.pipe(map(data => {
+        if (Object.keys(data).length !== 0 ) {
+          console.log(data);
+          return this.emailService.composeRequestAccessEmail(this.tool, data.imgProvider);
+        } else {
+          return this.emailService.composeRequestAccessEmail(this.tool, 'undefined');
```

Comment:
I don't think 'undefined' should be part of the message.

Reply:
I was actually not sure what to put in. This line is there so just in case imgProvider is not defined, there will be a string, and we know this case is handled properly. 
Because I believe the message we had before was also "undefined", but we didn't know what was missing